Makefile

```diff
@@ -19,8 +19,9 @@ LD_FLAGS = -ldflags " \
 			-X 'github.com/percona/pmm-managed/vendor/github.com/percona/pmm/version.Branch=$(PMM_RELEASE_BRANCH)' \
 			"
 
-release:                        ## Build pmm-managed release binary.
+release:                        ## Build pmm-managed and pmm-managed-init release binaries.
 	env CGO_ENABLED=0 go build -v $(LD_FLAGS) -o $(PMM_RELEASE_PATH)/pmm-managed
+	env CGO_ENABLED=0 go build -v $(LD_FLAGS) -o $(PMM_RELEASE_PATH)/pmm-managed-init ./cmd/pmm-managed-init
```

Comment:
you need to add changes to https://github.com/percona/pmm-server-packaging/blob/PMM-2.0/rhel/SPECS/pmm-managed.spec because now only pmm-managed binary is installed

Reply:
Yep, I know. Now you know too. :-)

Reply:
@Sevlin please have a look at it and prepare needed changes.

Reply:
Updated RPM spec, created PR.